.github/tests/tanzu-java-web-app-1/assertions.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euxo pipefail
+
+pushd $1
+./mvnw clean package
```

Comment:
No need to clean

Should we try building (sometimes with gradle) on more projects?

Reply:
Yes, I'll update it to build all the Java projects.